Audits Bridges (#1204)
maticnetwork/matic-docs: docs/develop/ethereum-polygon/plasma/eth.md

### process.env

> Create a new file in root directory named process.env, with the following content:

Reviewer: ```suggestion
Create a new file in the root directory named `process.env`, with the following content:
```

MATIC_RPC =
```

## Deposit

> **deposit**: Deposit can be done by calling **_depositEther_** on depositManagerContract contract.

Reviewer: ```suggestion
**deposit**: Deposit can be done by calling `depositEther()` on `depositManagerContract` contract.
```

### 1. Burn

> User can call the **_withdraw_** function of **_getERC20TokenContract_** child token contract. This function should burn the tokens. Polygon Plasma client exposes **_withdrawStart_** method to make this call.

Reviewer: ```suggestion
Users can call the `withdraw` function of `getERC20TokenContract` child token contract. This function should burn the tokens. Polygon Plasma client exposes `withdrawStart` method to make this call.
```

### 2. confirm-withdraw.js

> User can call the **_startExitWithBurntTokens_** function of **_erc20Predicate_** contract. Polygon Plasma client exposes the **_withdrawConfirm_** method to make this call. This function can be called only after the checkpoint is included in the main   chain. The checkpoint inclusion can be tracked by following this [guide](/docs/develop/ethereum-polygon/plasma/deposit-withdraw-event-plasma#checkpoint-events).

Reviewer: ```suggestion
Users can call the `startExitWithBurntTokens()` function of `erc20Predicate` contract. Polygon Plasma client exposes the `withdrawConfirm()` method to make this call. This function can be called only after the checkpoint is included in the main chain. The checkpoint inclusion can be tracked by following this [guide](/docs/develop/ethereum-polygon/plasma/deposit-withdraw-event-plasma.md#checkpoint-events).
```

### 3. Process Exit

> A user should call the **_processExits_** function of **_withdrawManager_** contract and submit the proof of burn. Upon submitting valid proof, tokens are transferred to the user. Polygon Plasma client exposes **_withdrawExit_** method to make this call.

Reviewer: ```suggestion
A user should call the `processExits()` function of the `withdrawManager` contract and submit the proof of burn. Upon submitting valid proof, tokens are transferred to the user. Polygon Plasma client exposes `withdrawExit` method to make this call.
```
